src/component/SortedLists.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { FirestoreCollection, withFirestore } from 'react-firestore';
+import dayjs from 'dayjs';
+
+const SortedList = ({ token, handlePurchase, calculateIfPurchased }) => {
+  const concatPath = `/lists/${token}/items`;
+  const now = new Date();
+  const today = dayjs(now);
+
+  return (
+    <section className="listFrame">
+      <h1>&#128722;My Shopping List</h1>
```

Comment:
Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria labelledby - to clear console warning.